src/main/java/in/yazhini/service/CustomerSelectBooks.java

```diff
@@ -0,0 +1,51 @@
+package in.yazhini.service;
+
+public class CustomerSelectBooks {
+	// Store the BookName in Array
+	public static boolean validateBookName(String bookName) {
```

Comment:
You are not validate the book name you are only checking the book name that will be present in the array or not so please change your method name to checkBookName.